mappings/net/minecraft/inventory/Inventories.mapping

```diff
@@ -1,4 +1,14 @@
 CLASS net/minecraft/class_1262 net/minecraft/inventory/Inventories
+	METHOD method_29234 remove (Lnet/minecraft/class_1263;Ljava/util/function/Predicate;IZ)I
+		ARG 0 inventory
+		ARG 1 shouldRemove
+		ARG 2 maxCount
+		ARG 3 dryRun
```

Comment:
I have no idea what `dryRun` does. Javadoc would be nice on the parameter

Reply:
Hmm, don't people know that a dry run is a simulation run? I am surprised that you haven't heard of the term.

Reply:
I know the concept, just not what it does in this context